Commands/CLI/bigmessage.js

```diff
@@ -0,0 +1,5 @@
+module.exports = async ({ cli }, cmdData, args) => {
+	args = args.trim();
+	if (args.length === 0) return winston.error("No input for BigMessage");
```

Comment:
`winston.warn` as error is used for critical erros

Reply:
except that is winston.error, not warn

Reply:
I said change it to `winston.warn` not it is winston.warn` rn